apps/els_core/src/els_core.app.src

```diff
@@ -0,0 +1,25 @@
+{application, els_core, [
+    {description, "Erlang Language Server Framework"},
+    {vsn, "0.1.0"},
+    {registered, []},
+    {applications, [
+        kernel,
+        stdlib,
+        ranch,
+        jsx,
+        cowlib,
+        yamerl,
+        redbug,
+        ephemeral
```

Comment:
Nit: I believe `ephemeral` is also only used by LSP (in the refactoring provider).

Reply:
Fixed as a follow up.